### ISSUE TITLE
[1.9.42] iPhone 5 s / 9.3.5, iPhone 6 S/ 10.0.2- "In progress" and "Lomotifs" tabs are not functional.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#911]
Frequency: Every time.

Precondition: There should exist at least one saved project.

Actions: 
1: Launch the application.
2: Swipe up to reveal profile view.
3: Check if "In progress" and "Lomotifs" tabs are functional. 

Expected result: "In progress" and "Lomotifs" tabs should be functional. 

Actual result: "In progress" and "Lomotifs" tabs are not functional.

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.9.42] on iPhone 6 / iOS 10.1.1

**Comment by Dipamit** [comment]
@xiuie : Able to reproduce this issue on iPhone 5/10.1.1 for build 1.9.42

**Comment by xiuie** [comment]
Are you signed in? Lomotifs tab should only be functional if you sign in.

**Comment by testershub** [comment]
@xiuie Yes, both "In Progress" and  "Lomotifs" tabs are not functional after sign in.
